src/storage/heap_file.c

```diff
@@ -22486,6 +22771,12 @@ heap_insert_logical (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context,
       goto error;
     }
 
+/*SUPPLEMENT_INSERT REDO LSA, INSERT REC_BIGONE  */
```

Comment:
Please, check all comments you wrote.

1. /*comment */ -> /\* comment */
2. there is one more space in the middle of the comments.
3. there is not enough explanation. So I was confused about whether it is needed or not.
4. there are some capital letters in the middle of the comments.

I hope you only leave comments that you really need and they include enough information.